ci-operator/step-registry/openshift/svt/reliability-v2/openshift-svt-reliability-v2-commands.sh

```
@@ -18,16 +18,19 @@ if [[ $REMOTE_CLIENT = "true" ]]; then
 SLACK_API_TOKEN=$(cat "/token/reliability-v2-slack-api-token")
 export SLACK_API_TOKEN
+set -x
 git clone https://github.com/openshift/svt --depth=1
 pushd svt/reliability-v2/utils
 git clone https://github.com/cloud-bulldozer/performance-dashboards.git --depth=1
 popd
 pushd svt/reliability-v2
 echo "========Start Reliability-v2 test for $RELIABILITY_DURATION========"
 set +e
-bash -x ./start.sh -n reliability -t $RELIABILITY_DURATION -c $CONFIG_TEMPLATE -r $TOLERANCE_RATE
+bash ./start.sh -n reliability -t $RELIABILITY_DURATION -c $CONFIG_TEMPLATE -r $TOLERANCE_RATE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)